entries/b/bookbeat.fi.json

```
@@ -0,0 +1,17 @@
+{
+  "BookBeat": {
+    "domain": "bookbeat.fi",
+    "url": "https://www.bookbeat.fi/",
+    "contact": {
```

```suggestion
    "additional-domains": [
      "bookbeat.at",
      "bookbeat.be",
      "bookbeat.com",
      "bookbeat.dk",
      "bookbeat.fr",
      "bookbeat.de",
      "bookbeat.it",
      "bookbeat.nl",
      "bookbeat.no",
      "bookbeat.pl",
      "bookbeat.es",
      "bookbeat.se",
      "bookbeat.ch",
      "bookbeat.co.uk"
    ],
    "contact": {
```

BookBeat has regional sites as well